app/models/manageiq/providers/ansible_tower/configuration_manager/configuration_script.rb

```diff
@@ -8,7 +8,7 @@ def run(vars = {})
   end
 
   def merge_extra_vars(external)
```

Comment:
For AnsibleTower service provisioning, yes the external contains all `extra_vars`. But as a public method in this class, the merge step is still recommended.